r/NEWS.md

```diff
@@ -19,6 +19,92 @@
 
 # arrow 10.0.1.9000
 
+## New features
+
+### Docs
+
+* A substantial reorganisation, rewrite of and addition to, many of the 
+  vignettes and README. (@djnavarro, 
+  [#14514](https://github.com/apache/arrow/issues/14514))  
```

Comment:
Simple question: Improved documentation is new features?

Reply:
It's substantial enough to warrant inclusion, yep.